features/eolearn/features/bands_extraction.py

```diff
@@ -0,0 +1,41 @@
+"""
+A collection of bands extraction EOTasks
+"""
+
+import numpy as np
+
+from eolearn.core import EOTask
+
+class EuclideanNormTask(EOTask):
+    """ The task calculates the Euclidean Norm:
+
+        :math:`\sqrt{\sum_{i} B_i^2}`,
```

Comment:
Huh, I noticed that pylint is complaining about \ signs here. For some weird reason Travis CI doesn't fail because of that.
https://travis-ci.org/sentinel-hub/eo-learn/jobs/562580382#L1229